### PR TITLE
typescript-angular: Ng43 with interfaces petstore scripts

### PR DESCRIPTION
**Pull request by feikede**
Added petstore sample scripts and output for typescript-angular 4.3 withInterfaces generation

### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [x] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

@macjohnny Added scripts to generate the petstore examples for angular-typescript 4.3 like discussed in https://github.com/swagger-api/swagger-codegen/pull/7975#issuecomment-381109842

Also added the example output.

### CODE REVIEW

**Comment by feikede** [comment]
@macjohnny Hi Esteban, sorry for the delay. I've created the 4.3 scripts now and added the example output. Let me know if there's anything more to do.

Thanks, Rainer

File: bin/typescript-angular-v4.3-petstore-interfaces.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    SCRIPT=`dirname "$SCRIPT"`/"$link"
+  fi
+done
+
+if [ ! -d "${APP_DIR}" ]; then
+  APP_DIR=`dirname "$SCRIPT"`/..
+  APP_DIR=`cd "${APP_DIR}"; pwd`
+fi
+
+executable="./modules/swagger-codegen-cli/target/swagger-codegen-cli.jar"
```
Comment (macjohnny) [inline bin/typescript-angular-v4.3-petstore-interfaces.sh]: could you please also add a script with interfaces for the newly added `v5`?

**Comment by macjohnny** [comment]
cc @TiFu (2017/07) @taxpon (2017/07) @sebastianhaas (2017/07) @kenisteward (2017/07) @Vrolijkx (2017/09) @macjohnny (2018/01)

**Comment by macjohnny** [comment]
cc @JohannesHoppe

**Comment by JohannesHoppe** [comment]
👍 Interfaces are a nice feature, especially in my NgRx world. I could test this branch with our code-base on Wednesday..

**Comment by feikede** [comment]
@macjohnny , @JohannesHoppe Hi all, is there anything more to do here?

**Pull request merged**